tests/managers/file/FileManagerTests.js

```diff
@@ -468,5 +468,58 @@ module.exports = function (appClient) {
             Validator.assertTrue(fileManager.hasUploadedFile(file));
             sdlManager.removeRpcListener(SDL.rpc.enums.FunctionID.ListFiles, expectSuccess);
         });
+
+        it('testCrcUndefinedVersion4', async function () {
+            const stub = sinon.stub(lifecycleManager, 'getSdlMsgVersion')
+                .callsFake(() => {
+                    return new SDL.rpc.structs.SdlMsgVersion()
+                        .setMajorVersion(4)
+                        .setMinorVersion(5);
+                });
+
+            const file = new SDL.manager.file.filetypes.SdlFile()
+                .setName('hello')
+                .setFilePath('./test_icon_1.png')
+                .setType(SDL.rpc.enums.FileType.GRAPHIC_PNG)
+                .setPersistent(true);
+
+            const putFile = await fileManager._createPutFile(file);
+
+            Validator.assertNull(putFile.getCrc());
```

Comment:
This test is failing for me. It seems that there isn't anything in the FileManager that would check the SdlMsgVersion before creating the CRC, so it won't be null here.